packages/udev/pubspec.yaml

```diff
@@ -8,6 +8,7 @@ environment:
 
 dependencies:
   ffi: ^1.0.0
+  meta: ^1.8.0
```

Comment:
this needs to be downgraded to `1.7.0` to be compatible with the latest flutter stable
```
Because every version of udev from path depends on meta ^1.8.0 and every version of integration_test from sdk depends on meta 1.7.0, udev from path is incompatible with integration_test from sdk.
```

Reply:
✅